--- grive-27817e835fe115ebbda5410ec904aa49a2ad01f1/cmake/Modules/FindJSONC.cmake~	2013-05-02 18:40:04.000000000 +0200
+++ grive-27817e835fe115ebbda5410ec904aa49a2ad01f1/cmake/Modules/FindJSONC.cmake	2013-12-04 01:11:58.326961184 +0100
@@ -6,8 +6,8 @@
 #  JSONC_INCLUDE_DIR - The JSON-C include directory.
 #  JSONC_LIBRARY     - The JSON-C library to link against.
 
-FIND_PATH(JSONC_INCLUDE_DIR json/json.h)
-FIND_LIBRARY(JSONC_LIBRARY NAMES json)
+FIND_PATH(JSONC_INCLUDE_DIR json-c/json.h)
+FIND_LIBRARY(JSONC_LIBRARY NAMES json-c)
 
 IF (JSONC_INCLUDE_DIR AND JSONC_LIBRARY)
    SET(JSONC_FOUND TRUE)
--- grive-27817e835fe115ebbda5410ec904aa49a2ad01f1/libgrive/src/protocol/Json.cc~	2013-05-02 18:40:04.000000000 +0200
+++ grive-27817e835fe115ebbda5410ec904aa49a2ad01f1/libgrive/src/protocol/Json.cc	2015-02-18 18:11:40.664989070 +0100
@@ -29,8 +29,8 @@
 	#pragma warning(push)
 	#pragma warning(disable: 4005)
 #endif
-#include <json/json_tokener.h>
-#include <json/linkhash.h>
+#include <json-c/json_tokener.h>
+#include <json-c/linkhash.h>
 #ifdef _MSC_VER
 	#pragma warning(pop)
 #endif
